Gemfile.lock

```diff
@@ -108,49 +108,49 @@ GEM
     mimemagic (0.3.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
     msgpack (1.3.3)
     nio4r (2.5.4)
     nokogiri (1.10.10)
       mini_portile2 (~> 2.4.0)
     popper_js (1.16.0)
-    public_suffix (4.0.5)
+    public_suffix (4.0.6)
     puma (4.3.6)
       nio4r (~> 2.0)
     rack (2.2.3)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.0.3.2)
-      actioncable (= 6.0.3.2)
-      actionmailbox (= 6.0.3.2)
-      actionmailer (= 6.0.3.2)
-      actionpack (= 6.0.3.2)
-      actiontext (= 6.0.3.2)
-      actionview (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    rails (6.0.3.4)
+      actioncable (= 6.0.3.4)
+      actionmailbox (= 6.0.3.4)
+      actionmailer (= 6.0.3.4)
+      actionpack (= 6.0.3.4)
+      actiontext (= 6.0.3.4)
+      actionview (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activemodel (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       bundler (>= 1.3.0)
-      railties (= 6.0.3.2)
+      railties (= 6.0.3.4)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    railties (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rake (13.0.1)
     rb-fsevent (0.10.4)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    regexp_parser (1.7.1)
+    regexp_parser (1.8.2)
```

Comment:
### [regexp_parser](https://github.com/ammar/regexp_parser)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.7.1 → 1.8.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/ammar/regexp_parser/compare/v1.7.1...v1.8.2).

These are the first 10 commits:
- (6085eb5) [Allow no-whitespace and single-line comments (#66)](https://github.com/ammar/regexp_parser/commit/6085eb5ed8ff1929ee9b71e3ebdd37860df80676)
- (58a00d6) [Merge pull request #67 from owst/allow_no_whitespace_comment…](https://github.com/ammar/regexp_parser/commit/58a00d6ff10a300876685742acb858accb6dc21b)
- (74856d9) [Let travis build all branches](https://github.com/ammar/regexp_parser/commit/74856d9762ca35e210ec9354b9c94523dfc18fd2)
- (84b5650) [Support \g, \k without group id as literal escapes](https://github.com/ammar/regexp_parser/commit/84b5650ab239309f382d5850a78c3fc48f08ade3)
- (7c4d268) [Allow options to be passed when processing a String pattern](https://github.com/ammar/regexp_parser/commit/7c4d2685a0190a4e944f3b53816692f17ff6361f)
- (eea67dd) [fixup! Allow options to be passed when processing a String p…](https://github.com/ammar/regexp_parser/commit/eea67dd9f4c8572e0e2adc1bfbe2aef60fce2a7a)
- (d109b56) [Merge pull request #68 from owst/allow_options_when_parsing_…](https://github.com/ammar/regexp_parser/commit/d109b568c32e1c4fb471e8078dda656785bb7a6b)
- (bae4ea0) [Release v1.8.0](https://github.com/ammar/regexp_parser/commit/bae4ea07b2b7574e615e112834e311daabf6334e)
- (1808959) [Specify correct minimum ruby after 7c4d268](https://github.com/ammar/regexp_parser/commit/1808959b4cf3c07c72f558b00f36d0398863e7ff)
- (1d36045) [Fix scanning of comment-like text in normal mode ...](https://github.com/ammar/regexp_parser/commit/1d36045798b2c068fd36081a6d88b1f0900911dc)


</details>